### DISCUSSION TITLE
`rain deploy` Is it possible to "force" rain to not modify the CloudFormation yaml?

### DISCUSSION THREAD

**Discussion by r-heimann** [aws-cloudformation/rain#326] [Q&A]
See title, is this actually possible?

**Comment by ericzbeard** [comment]
Not currently. It might be a feature we could add, but it would only be possible for templates that don't require any packaging.

**Reply by r-heimann** [reply]
It would be a really great feature to have!
It would certainly decrease the amount of time you need to invest to debug templates, since what you wrote actually is being used by CloudFormation to deploy your infrastructure. This often happens with CDK, since you can't be 100% sure that the CloudFormation produced by CDK is causing an issue with CloudFormation.
